src/utilities/filetypes/RunOptions.hpp

```diff
@@ -100,9 +110,13 @@ class UTILITIES_API RunOptions
   bool setCustomOutputAdapter(const CustomOutputAdapter& adapter);
   void resetCustomOutputAdapter();
 
-  std::string forwardTranslateOptions() const;
-  bool setForwardTranslateOptions(const std::string& options);
-  void resetForwardTranslateOptions();
+  OS_DEPRECATED std::string forwardTranslateOptions() const;
+  OS_DEPRECATED bool setForwardTranslateOptions(const std::string& options);
+  OS_DEPRECATED void resetForwardTranslateOptions();
+
+  ForwardTranslatorOptions forwardTranslatorOptions() const;
+  bool setForwardTranslatorOptions(const ForwardTranslatorOptions& forwardTranslatorOptions);
+  void resetForwardTranslatorOptions();
```

Comment:
Deprecated and new. Before it was storing everything as a json encoded string, now it's an explicit class.

Deprecated are kept for compat with ruby workflow-gem until it gets phased out